tests/circuit_drawer/test_circuit_drawer.py

```diff
@@ -93,7 +93,7 @@ def to_layer(operation_list, num_wires):
     layer = [None] * num_wires
```

Comment:
Just commenting here but really I have a question about another file:
@josh146 is the file `qchem/tmp1v5rrgvd` something that is needed in version control?

Reply:
Nope, wonder where it came from 🤔